components/core/src/Core.tsx

```diff
@@ -5,13 +5,12 @@ import { Fragment } from 'react';
 import { useBrand } from './Brand';
 import { normalize as normalizeCSS } from './normalize';
 
-export interface ICoreProps {
+export interface CoreProps {
```

Comment:
Shouldn't this interface be called "ICoreProps"?
```suggestion
export interface ICoreProps {
```